pkg/storage/pebble.go

```diff
@@ -129,6 +130,24 @@ var MinCapacityForBulkIngest = settings.RegisterFloatSetting(
 	0.05,
 )
 
+// readaheadModeInformed controls the pebble.ReadaheadConfig.Informed setting.
+var readaheadModeInformed = settings.RegisterIntSetting(
+	settings.ApplicationLevel, // used by temp storage as well
+	"storage.readahead_mode.informed",
+	"the readahead mode for operations which are known to read through large chunks of data (0=off, 1=SYS_READAHEAD, 2=FADV_SEQUENTIAL)",
+	2,
+	settings.IntInRange(0, 2),
+)
+
+// readaheadModeSpeculative controls the pebble.ReadaheadConfig.Speculative setting.
+var readaheadModeSpeculative = settings.RegisterIntSetting(
+	settings.ApplicationLevel, // used by temp storage as well
+	"storage.informed_readahead_mode.speculative",
```

Comment:
nit: `storage.readahead_mode.speculative`?

Or is "speculative informed readahead mode" a thing?